pkg/apis/workflow/v1alpha1/generated.proto

```diff
@@ -138,6 +159,9 @@ message ArtifactRepositoryRefStatus {
 
   // If this ref represents the default artifact repository, rather than a config map.
   optional bool default = 3;
+
+  // The repository the workflow will use. This maybe empty before v3.1.
```

Comment:
Should this be v3.2 since v3.1 won't have this feature?

Reply:
I plan to backport.